pkg/kv/kvserver/store.go

```diff
@@ -1743,6 +1744,15 @@ func NewStore(
 	queueAdditionOnSystemConfigUpdateBurst.SetOnChange(&cfg.Settings.SV,
 		updateSystemConfigUpdateQueueLimits)
 
+	// Set up callback to update lock table size when the cluster setting changes.
```

Comment:
I think the function names are clear enough here to elide this comment if you wan.